app/src/pages/explorer/curate/validlist/UpdateXlsList.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div class="spreadsheet_list_form section_teams">
       <div >
+          <div style="max-width: 99%;">
+              <div class="md-layout md-alignment-top-center">
+                  <div class="md-layout-item md-size-50 md-medium-size-70 md-small-size-85 md-xsmall-size-95">
+                    <CurateNavBar active="" :navRoutes="[]"/>
```

Comment:
Can we put this `[]` in data? I noticed you already have a default value in data

Reply:
Good catch, I missed this